src/components/connection_handler/include/connection_handler/connection_handler_impl.h

```diff
@@ -646,7 +649,14 @@ class ConnectionHandlerImpl
    **/
   void RemoveConnection(const ConnectionHandle connection_handle);
 
-  void OnConnectionEnded(const transport_manager::ConnectionUID connection_id);
+  /**
+   * @brief Called when connection is closed.
+   *
+   * @param connection_id Connection unique identifier.
+   * @param close_reason Reason of the session close
```

Comment:
```suggestion
   * @param close_reason Reason for a session closing
```

Reply:
@IGapchuk 
Fixed in:https://github.com/LuxoftSDL/sdl_core/pull/136/commits/957e39c33032718558882f177d2cb17a44624afc